test/src/index.test.js

```diff
@@ -37,6 +37,18 @@ describe('Anonymizer', () => {
       });
     });
 
+    it(`should obfuscate keys which type is Buffer`, () => {
+      const anonymize = anonymizer();
+
+      expect(anonymize({ foo: Buffer.from('foobarfoobar') })).toEqual({ foo: '--REDACTED--' });
+    });
+
+    it(`should not obfuscate keys which type is Buffer and are whitelisted`, () => {
```

Comment:
Maybe clearer this way:

```suggestion
    it(`should not obfuscate Buffer-type keys that are whitelisted`, () => {
```